packages/main/src/Dialog.js

```diff
@@ -267,6 +300,116 @@ class Dialog extends Popup {
 		});
 		this.removeEventListener("ui5-before-close", this._recenter);
 	}
+
+	_onResizeMouseDown(event) {
+		if (!(this.resizable && this.onDesktop)) {
+			return;
+		}
+
+		event.preventDefault();
+
+		const {
+			top,
+			left,
+		} = this.getBoundingClientRect();
+		const {
+			width,
+			height,
+			minWidth,
+			minHeight,
+		} = window.getComputedStyle(this);
+
+		this._initialX = event.clientX;
+		this._initialY = event.clientY;
+		this._initialWidth = Number.parseFloat(width);
```

Comment:
Please use the global function ```parseFloat``` as it is natively supported by IE. This way babel would not have to transform it when it comes to the es5 build

Reply:
I have added a seperate pull request to add support for Number.parseFloat https://github.com/SAP/ui5-webcomponents/pull/2302